packages/samples/sudt/src/actors/sudt.model.ts

```diff
@@ -0,0 +1,152 @@
+/**
+ * @module src/actors/omnilock.model
+ * @description
+ * This is the actor model for omnilock, which is used to gather omnilock cells to generate record models.
+ */
+
+import {
+  ActorProvider,
+  Param,
+  ActorReference,
+  CellPattern,
+  JSONStore,
+  OutPointString,
+  SchemaPattern,
+  UpdateStorageValue,
+  TypeFilter,
+  Sudt,
+} from '@ckb-js/kuai-models'
+import type { Cell, HexString, Script } from '@ckb-lumos/base'
+import { number, bytes } from '@ckb-lumos/codec'
+import { InternalServerError } from 'http-errors'
+import { BI, utils, config } from '@ckb-lumos/lumos'
+import { MIN_SUDT_WITH_OMINILOCK, TX_FEE } from '../const'
+
+/**
+ * add business logic in an actor
+ */
+@ActorProvider({ ref: { name: 'sudt', path: `/:lockArgs/:typeArgs/` } })
+@TypeFilter()
+@Sudt()
+export class SudtModel extends JSONStore<Record<string, never>> {
+  constructor(
+    @Param('args') args: string,
```

Comment:
```suggestion
    @Param('lockArgs') lockArgs: string,
    @Param('typeArgs') typeArgs: string,
```

Reply:
This has been fixed by https://github.com/ckb-js/kuai/pull/509